src/dst/bind.cpp

```diff
@@ -1148,6 +1150,137 @@ static std::unique_ptr<Expr> fracture(std::unique_ptr<Top> top) {
     }
   }
 
+  for (auto &package : top->packages) {
+    for (auto &file : package.second->files) {
+      std::map<std::string, std::pair<int, Location>> imports = {};
```

Comment:
You should document what the pair does or replace it with a small struct. You can define a struct inline inside this function I think. It wasn't obvious to me that the integer was a use count.